### PR TITLE
Release ALBs!

### PR DESCRIPTION
**Pull request by krallin**
This shows the platform for a given Endpoint. We shouldn't release this
until ALB is actually the new default in API.
## 

Screenshot (happy to tweak the copy):

image

cc @fancyremarker @sandersonet @gib

### CODE REVIEW

**Comment by fancyremarker** [comment]
👍 , explanation looks good.

**Comment by krallin** [comment]
I'm upgrading this PR a little bit. Rather than just show the platform, customers can now pick their platform... and upgrade to ALB! This uses the new fancy modals with a little help from @sandersonet 😄 

Screenshots:

Endpoint List:

image

Upgrade modal:

image

New Endpoint menu (this is starting to be a little crowded 😢 ):

image

File: app/app/vhosts/new/route.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default Ember.Route.extend({
     var stack = app.get('stack');
 
     return Ember.RSVP.hash({
-      vhost: this.store.createRecord('vhost', { app }),
+      vhost: this.store.createRecord('vhost', { app, platform: 'alb' }),
```
Comment (sandersonet) [inline app/app/vhosts/new/route.js]: If they all default to `alb` now, we could just set this as the `defaultValue` for the `platform` attr directly on the model.

Comment (krallin) [inline app/app/vhosts/new/route.js]: Nice; thanks! Updated.

**Pull request merged**